clients/privacy-center/packages/fides-consent/rollup.config.js

```diff
@@ -23,8 +23,13 @@ const generateConsentConfig = () => {
    * @type {import('../../types/config').Config}
    */
   const privacyCenterConfig = require("../../config/config.json");
-  const privacyCenterOptions =
-    privacyCenterConfig.consent?.consentOptions ?? [];
+  const isV1ConsentConifg = typeof privacyCenterConfig.consent?.button === "undefined";
```

Comment:
is it not possible to use the `isV1ConsentConfig` function you made earlier, here? 

I wouldn't be surprised if not, since this is a JS file and not a TS one, but I just want to make sure

Reply:
I made the assumption here that `fides-consent.js` may not always be guaranteed to have access to the `isV1ConsentConfig` function given that this is its own package — even though it does currently.